README.rst

```diff
@@ -24,22 +24,19 @@ f5-openstack-lbaasv2-driver
 Introduction
 ------------
 
-The F5® OpenStack LBaaSv2 service provider driver (also called the 'F5® LBaaSv2 driver') makes it possible to provision F5® BIG-IP® local traffic management (LTM®) services in an OpenStack cloud.
+The F5® OpenStack LBaaSv2 service provider driver (also called the 'F5 LBaaSv2 driver') makes it possible to provision F5 BIG-IP® Local Traffic Manager® (LTM®) services in an OpenStack cloud.
 
-The F5® LBaaSv2 driver works in conjunction with the F5® OpenStack agent, which uses the :ref:`f5-sdk <f5sdk:home>` to translate Neutron API calls into BIG-IP® REST API calls.
+The F5 LBaaSv2 driver works in conjunction with the F5® OpenStack agent, which uses the `f5-sdk <http://f5-sdk.readthedocs.io/en/latest/>`_ to translate Neutron API calls into BIG-IP REST API calls.
```

Comment:
Up for debate. I think it's enough to just link to the doc set and latest is the easiest to link to.